plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -833,6 +978,27 @@ public static Object of(
         }
     }
 
+    @Override
+    @Test(enabled = false)
+    public void testShowCreateTable() {}
```

Comment:
Wh did you disable the test? We shouldn't do that. Please fix `matches` string in this class.

Reply:
I thought if that is not supported, then there was no point in running those. Ideally, the whole reason for enabled = false is these tests won't be picked up in the running and it's a better and faster build.

I will match this as not supported if it's essential to maintain that.